auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -90,13 +90,18 @@
 from monai.metrics import DiceHelper
 
 
+from monai.apps.utils import DEFAULT_FMT
+from monai.apps.auto3dseg.auto_runner import logger
+print = logger.debug
```

Comment:
Will there be any info printed out if the level is set to "info"?

Reply:
currently, we only use logger.debug inside of the algo, so nothing is printed in info